.github/workflows/build-image.yml

```diff
@@ -97,8 +97,7 @@ jobs:
             path: "github-actions"
             exportToken: true
             secrets: |
-              secret/data/github-actions-common/github/read-repo-token USERNAME | GH_ORG_READ_REPO_USER ;
-              secret/data/github-actions-common/github/read-repo-token PASSWORD | GH_ORG_READ_REPO_TOKEN ;
+              github/token/renovate-renovatebot token | GH_ORG_READ_REPO_TOKEN ;
```

Comment:
Why use renovate token, just for simplicity because it has the same permissions needed here ?

Reply:
parce que je suis con et que je fais des copier coller